inst/rmarkdown/templates/science/skeleton/skeleton.Rmd

```diff
@@ -0,0 +1,210 @@
+---
+title: Template for *Science* Articles
+authors:
+  - name: V. O. Key
+    affiliation: 1
+    corresponding_author: true
+  - name: Sidney Verba
+    affiliation: "1,2" # (multiple affiliations shold be quoted and separated by a comma)
+  - name: Marvin Zelen
+    affiliation: 3
+  - name: Robert Dahl
+    affiliation: 4
+address:
+  - code: 1
+    address: Harvard University, Department of Government, Cambridge, MA 02138
+  - code: 2
+    address: Harvard University, Institute for Quantitative Social Science, Cambridge, MA 02138
+  - code: 3
+    address: Harvard University, Department of Biostatistics, Cambridge, MA 02138
+  - code: 4
+    address: Yale University, Department of Political Science, Yale, CT 06520
+corresponding_author: "To whom correspondence should be addressed; E-mail: key@harvard.edu."
+abstract: |
+  Enter the text of your abstract here.
+acknowledgements: |
+  Please include your acknowledgments here, set in a single paragraph.
+bibliography: scibib.bib
+output: 
+  rticles::science_article
+---
+
+# Introduction
+
+This is a basic implementation of the Science template. 
+
+# Uniform Distributions 
+
+Uniform distributions text with Figure.
+
+```{r once, echo=FALSE, fig.cap='test once', fig.align='center', out.width='50%'}
+plot(runif(10))
+```
+
+And post uniform distributions text...
+
+\begin{table}[ht]
+\begin{center}
+\begin{tabular}{ll}
+once & 1 \\
+once & 2\\
+\end{tabular}
+\end{center}
+\end{table}
+
+With a table.
+
+# Poisson Distributions
+
+Poisson distributions text...
+
+```{r twice, echo=FALSE, fig.cap='test twice'}
+plot(rpois(10,1))
+```
+
+And post Poisson distributions text...
+
+\begin{table}[ht]
+\begin{center}
+\begin{tabular}{ll}
+twice & 1 \\
+twice & 2\\
+\end{tabular}
+\end{center}
+\end{table}
+
+With a table for something.
+
+# Formatting Citations
+
+\begin{enumerate}
+\item Please cite your references and notes in text {\it only\/} using
+  the standard \LaTeX\ \verb+\cite+ command, not another command
+  driven by outside macros.
+\item Please separate multiple citations within a single \verb+\cite+
+  command using commas only; there should be {\it no space\/}
+  between reference keynames.  That is, if you are citing two
+  papers whose bibliography keys are \texttt{keyname1} and
+  \texttt{keyname2}, the in-text cite should read
+  \verb+\cite{keyname1,keyname2}+, {\it not\/}
+  \verb+\cite{keyname1, keyname2}+.
+\end{enumerate}
```

Comment:
Is there a reason to keep this all latex and not use Markdown list and other syntax ?

Just curious

Reply:
Nope, this is now updated.